src/journal/src/grpc/proto.rs

```diff
@@ -14,13 +14,12 @@
 
 tonic::include_proto!("engula.journal.grpc.v1");
```

Comment:
```suggestion
tonic::include_proto!("engula.journal.v1");
```

I think "engula.journal.v1" is sufficient, the "grpc" is redundant.

Reply:
OK.